src/explore-education-statistics-common/src/modules/table-tool/components/TableHeadersGroup.tsx

```diff
@@ -77,10 +86,15 @@ const TableHeadersGroup = ({
           <TableHeadersGroupControls
             defaultNumberOfItems={defaultNumberOfItems}
             groupName={name}
-            id={groupId}
+            id={id}
+            index={index}
+            isLastGroup={isLastGroup}
```

Comment:
If one were to be suuuuuuuuper pedantic, one might point out that we pull an `activeGroup` id out of the TableHeadersContext rather than an `isActiveGroup` flag, so we're introducing two different strategies for how we check for "is this the active group?" and "is this the last group?". 

Personally, I don't have the slightest problem with this as each one makes perfect sense in its context :D Just pointing it out...